lib/libutee/default_bold.c

```diff
@@ -0,0 +1,1009 @@
+/*
```

Comment:
I guess the license should be the license of the TTF used to generate this then.